unitex/src/fr/umlv/unitex/frames/GraphFrame.java

```diff
@@ -916,6 +916,8 @@ public void toDo(boolean success) {
                     if (line.length() < 2) {
                         continue;
                     }
+                    if (line.indexOf("#/{") > -1) //signature of graph B
```

Comment:
Braces are used with `if` even when the body is empty or contains only a single statement